contracts/src/v0.8/dev/ocr2dr/OCR2DROracle.sol

```diff
@@ -7,27 +7,40 @@ import "../AuthorizedReceiver.sol";
 import "../../ConfirmedOwner.sol";
 
 /**
- * @title OCR2DR oracle contract (stub for now)
+ * @title OCR2DR oracle contract
  */
 contract OCR2DROracle is OCR2DROracleInterface, AuthorizedReceiver, ConfirmedOwner {
   event OracleRequest(bytes32 requestId, bytes data);
   event OracleResponse(bytes32 requestId);
 
+  error EmptyRequestData();
   error InvalidRequestID();
-  error Unauthorized();
-  error NonceMustBeUnique();
+  error LowGasForConsumer();
 
   struct Commitment {
     address client;
     uint256 subscriptionId;
   }
 
+  uint256 private constant MINIMUM_CONSUMER_GAS_LIMIT = 400000;
+
   uint256 private s_nonce;
   mapping(bytes32 => Commitment) private s_commitments;
 
   constructor(address owner) ConfirmedOwner(owner) {}
 
+  /**
+   * @notice The type and version of this contract
+   * @return Type and version string
+   */
+  function typeAndVersion() external pure virtual returns (string memory) {
+    return "OCR2DROracle 1.0.0";
```

Comment:
Could we start at `0.0.0` for development release? (With 1.0.0 being the first production release)